hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestHSync.java

```diff
@@ -116,6 +119,9 @@ public static void init() throws Exception {
     CONF.setBoolean(OZONE_OM_RATIS_ENABLE_KEY, false);
     CONF.set(OZONE_DEFAULT_BUCKET_LAYOUT, layout.name());
     CONF.setBoolean(OzoneConfigKeys.OZONE_FS_HSYNC_ENABLED, true);
+    CONF.setBoolean("ozone.client.incremental.chunk.list", true);
```

Comment:
I had to make it false by default to pass compatibility tests.
Ideally, a new client would detect DataNode version and then fallback to full chunk list. However, due to https://issues.apache.org/jira/browse/HDDS-9884, this is currently not possible and compat test would fail.

I can revisit this after HDDS-9884 is fixed.

Reply:
Filed HDDS-10252 to track this task.